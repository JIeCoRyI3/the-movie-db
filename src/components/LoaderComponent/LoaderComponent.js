import React from 'react';
import styles from './LoaderComponent.module.css';

function LoaderComponent() {
    return (
        <div className={styles.ldsDualRingParent}>
            <div className={styles.ldsDualRing}></div>
        </div>
    );
}

export default LoaderComponent;
