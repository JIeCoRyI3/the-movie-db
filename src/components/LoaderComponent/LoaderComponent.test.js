import React from 'react';
import { shallow } from 'enzyme';
import LoaderComponent from './LoaderComponent';

describe('<LoaderComponent/> render', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<LoaderComponent />);
    });

    it('should render LoaderComponent component', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('should have div with class ldsDualRing', () => {
        expect(wrapper.find('.ldsDualRing')).toMatchSnapshot();
    });
});
