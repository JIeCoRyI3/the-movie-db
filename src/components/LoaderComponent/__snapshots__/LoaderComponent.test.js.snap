// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoaderComponent/> render should have div with class ldsDualRing 1`] = `null`;

exports[`<LoaderComponent/> render should render LoaderComponent component 1`] = `
<div>
  <div />
</div>
`;
